allauth/account/signals.py

```diff
@@ -1,9 +1,22 @@
-from django.dispatch import Signal
+from django.contrib.auth.signals import user_logged_out as dj_user_logged_out
+from django.dispatch import Signal, receiver
 
 
 user_logged_in = Signal(providing_args=["request", "user"])
 user_logged_out = Signal(providing_args=["request", "user"])
 
+
+@receiver(dj_user_logged_out, dispatch_uid="forwards_user_logged_out")
+def forwards_user_logged_out(sender, **kwargs):
```

Comment:
Wondering, is this really necessary? What would go wrong if you just imported the django signal as is, without doing any forwarding/intervention at all.